impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -49,14 +49,17 @@ static vk::AttachmentDescription CreateAttachmentDescription(
 
   if (desc.storage_mode == StorageMode::kDeviceTransient) {
```

Comment:
Is the real problem that we're setting the storage mode to device transient?

I'm not quite clear on when we should set dontcare or not though.

Reply:
no that doesn't seem to be it, at least things are already working for me